src/backend/marsha/core/templates/core/mail/html/sample.html

```diff
@@ -0,0 +1,214 @@
+<!doctype html>
```

Comment:
HTML and txt templates should not be commited. Only the `mjml`. Otherwise there is no need to compile them in the `Dockerfile` as they are already in the projet.